docs/git/cloning.md

```diff
@@ -99,33 +99,48 @@ If you haven't already, now is a good time to install the Zulip development envi
 source projects in general, we recommend following our [detailed guide for
 first-time contributors][zulip-rtd-dev-first-time].
 
-## Step 3: Configure Travis CI (continuous integration)
+## Step 3: Configure CircleCI and Travis CI (continuous integration)
```

Comment:
I think this title should be "Step 3: Configure continuous integration for your fork", since that's what it does (and that way the title doesn't need to focus on which tool(s) we're using; don't want to have to edit all the links if we later drop Travis).